### ISSUE TITLE
concept:intersects

### ISSUE THREAD

**Issue by nielsklazenga** [tdwg/tcs2#57]
### intersects (concept)

|            |                                                                |
| ---------- | -------------------------------------------------------------- |
| Label      | Intersects                                                     |
| Definition | The object and subject taxon concepts have at least one member in common |
| Comments   |                                                                |

**Comment by camwebb** [comment]
@nielsklazenga I think it's important to separate the nomenclatural terms (synonymOf) from TC relationship terms. A synonymOf B relates two names, not TCs. I know TCS1 used `synonym` as (defunct) TC relationship term, but this is one of very few places where I disagree with the TCS1 authors. In our TC mapping work for the Alaska flora we have two fields: TC relationship and synonym type.

**Comment by nielsklazenga** [comment]
@camwebb, you and me both. Synonyms have the same relationship to a Taxon Concept as the accepted name (`taxonName`).

I was merely wondering in what circumstances `intersects` would be used. I was thinking of old literature reports where you do not have reason to doubt the determination, but where the only information you have is the name, which is either the accepted name or a synonym of a Taxon Concept you currently recognise.

**Comment by deepreef** [comment]
I agree completely.  The concept of "synonym" is nomenclatural.  However, it does play a role in taxonomy, in that heterotypic synonyms impact taxon concept circumscriptions differently than homotypic.

But in any case, I think TCS2 needs to capture "synonymOf" at the granularity of individual TNUs (not names -- if names are maintained as separate entities from TNUs).

**Comment by camwebb** [comment]
@nielsklazenga Ah yes, I see what you were asking now. And yes, where no TC mapping has been done, but we do know that there is at least on specimen in common between two TCs (which we might indeed know from a synonym nomenclatural statement) at a minimum we can say that two TCs intersect. We find we use this term in a lot.  I agree with @nfranz's suggestion of adding an (A OR B OR ...) definition too, though perhaps 'intersects is a superclass of...' is maybe a bit clearer?

**Comment by nielsklazenga** [comment]
@deepreef, I agree with you too. It is the Taxon Concepts (which are TNUs) that have synonyms (at least the non-homotypic ones), not the Taxon Names. Your "synonymOf" is the same thing as `acceptedNameUsage` (#8), which I propose to name `taxonConcept`, so it can be used more generally, _e.g._ for vernacular names. (I thought I had already done that, but I guess not). I would also like a `synonyms` property on the Taxon Concept (TNU), so people can embed synonyms. BTW, as soon as you add an ID to the synonym assertion, the synonym itself is also a TNU.

@camwebb, I actually think @nfranz's formulas are the clearest, but we might just describe them in the usage notes and have a table somewhere with formulas that illustrate the relationships between all taxon relationship types. I think we should avoid any talk of superclasses here though.

**Comment by nielsklazenga** [comment]
One thing that came up in the discussion in the TNC IG meeting this morning is that people want another label for this term. Easier said than done, but we should think about it. I think part of the problem is in the term `overlaps`, as the words 'overlap' and 'intersect' essentially mean the same thing.

**Comment by camwebb** [comment]
Agree @nielsklazenga, the problematic term is `overlap`. Not really sure 'partially' is the right modifier (objectively), but since RCC-5 uses it, that may be the best way to go.

**Comment by deepreef** [comment]
The part that's a little confusing to me is that presumably the five standard relationships are mutually exclusive with respect to each other.  But "Intersects" could be the same as Congruent, or could be the same as Includes/IncludedIn, or could be the same as Overlaps (but it's definitely at least one of those).  In other words, Intersects would be the only one that is non-mutually-exclusive with the others (except Excludes), and the only one that is ambiguous in its logical representation.

Wouldn't an alternative be to keep the five standard ones as they are, then add an optional "not" parameter that could be applied to "Excludes" to represent the same thing as "Intercepts"?  I can imagine some value in applying "not" to others as well (e.g., not congruent, without being sure whether it's Includes/IncludedIn or Overlaps).

**Comment by nielsklazenga** [comment]
@camwebb , yes, I would have thought 'partly', but the meaning is almost exactly the same, which is why I just followed RCC-5.

@deepreef, I see your point. We can make this clear by using a taxonomy:

```
|- intersects
|   |- is congruent with
|   |- has proper part
|   |- is proper part of
|   |- partially overlaps
|- is disjoint to
```

We can do that in SKOS (which TDWG controlled vocabularies use) using skos:broader (Steve's scripts already take this into account). The problem might be that we want people to use `intersects` might see this as a divide between `intersects` and `is disjoint to`, while we really want them to use the other four terms.  So, what we really want to have might be something like this:

```
|- is congruent with
|- has proper part
|- is proper part of
|- partially overlaps
|- any of the above (intersects)
|- none of the above (is disjoint to)
```

I think that is exactly the same. We just have to find some other way to make clear which the more important terms are.

If we had a 'NOT' parameter, we obviously would not need `intersects` anymore, but that would break data exchange, as systems could ignore this parameter (we can obviously not make it required) and not exchange it, but still exchange the relationship type (which is required). If we want to use negation, negated terms need to be added to the controlled vocabulary, as has been done for e.g. [`notatype`](http://rs.gbif.org/vocabulary/gbif/type_status/notatype) (that is actually the only example I can think of).

We've got a clear use case for including `intersects`. If such a use case comes up for 'not congruent', we can consider that as well.

**Comment by nielsklazenga** [comment]
@camwebb: Could you provide an example of the use of this term from your work for the doc?
